src/Symfony/Component/VarDumper/Cloner/VarCloner.php

```diff
@@ -94,8 +94,11 @@ protected function doClone($var)
                 // Create $stub when the original value $v can not be used directly
                 // If $v is a nested structure, put that structure in array $a
                 switch (true) {
-                    case empty($v):
```

Comment:
`'0'` is also empty according to PHP

Reply:
true, but there is no need to make it a special case: handling it in the "is_string" case is enough IMHO